src/ER/Beds.js

```diff
@@ -75,17 +82,19 @@ function Beds() {
                 </>
               }
               property="additionalNotes"
-              value={additionalNotes}
+              value={additionalNotes == null ? statusUpdate.additionalServiceAvailabilityNotes : additionalNotes}
               onChange={(property, value) => setAdditionalNotes(value)}
             />
-            <div className="text-right">
-              {updatedNotes && (
-                <span className="beds__updated">
-                  <img className="beds__check-icon" src={CheckIcon} alt="check icon" width="25" height="30" />
-                  Updated
-                </span>
-              )}
-              <button className="usa-button" type="button" onClick={handleNotesUpdate}>
+            <div className="beds__notes-controls">
+              <span className="beds__updated">
+                {showNotesUpdated && (
+                  <>
+                    <i className="fas fa-check-circle" />
```

Comment:
@abmcbride5 just a note that I've had Font Awesome Free in our project since the beginning so that, whenever possible, we can use scalable vector icons from its webfont instead of adding new bitmap assets to the project. https://fontawesome.com/

Also check out the changes I've made to the HTML/CSS for styling and positioning this "Updated" alert that you added recently. I'm wondering how you decided to size things originally- you put in a font-size: 21px even though the Figma shows that the text is H4/16px, and you put a width/height of 25x30 for the icon even though it's 12x12 in Figma...?

Try not to use absolute positioning unless necessary- I'm really starting to enjoy using CSS Flexbox now, and it made it pretty easy/straightforward to get the arrangement in the design...